web/app/controllers/bloggs_controller.rb

```diff
@@ -12,17 +12,19 @@ def show
 
   def new
     @article = Blogg.new
+    authorize! :new, @article
   end
 
   def edit
     puts 'EDITING'
     puts params
     @article = Blogg.find(params[:id])
+    authorize! :edit, @article
```

Comment:
Looks like we're using `authorize!` on each method, why not use the `authorize_resource` as described here: https://github.com/ryanb/cancan/wiki/authorizing-controller-actions ?